yt_dlp/extractor/iwara.py

```diff
@@ -56,9 +65,9 @@ class IwaraIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage, urlh = self._download_webpage_handle(url, video_id)
+        webpage = self._download_webpage(url, video_id)
 
-        hostname = compat_urllib_parse_urlparse(urlh.geturl()).hostname
```

Comment:
This is using `urlh.geturl()` in order to account for any redirects. Change this only if you are sure this isn't needed